src/graphql/query-builder.ts

```diff
@@ -133,7 +137,11 @@ export class QueryBuilder {
       if (isNil(v)) {
         mutObj[k] = null;
       } else if (v instanceof Ref) {
-        mutObj[k] = this.deleteMutationObj(v.model, true);
+        if (v.model) {
+          mutObj[k] = this.deleteMutationObj(v.model, true);
+        } else {
+          mutObj[k + 'Id'] = {_is_null: true};
```

Comment:
why is this `k + 'Id'`?

Reply:
because references show up on the model as <field_name> + `Id`. So a where clause that is checking to make sure that the reference is null needs to check that <field_name> + `Id` _is_null: true